packages/config/src/configDefaults.ts

```diff
@@ -28,6 +28,10 @@ export const getInitialConfig = ({
       }
     },
     verboseRpc: false,
+    solidityLog: {
+      displayPrefix: "",
+      disableMigrate: false
```

Comment:
This has probably already been discussed, but I don't tend to not dislike :wink: boolean settings with a negative in the name... it's just confusing. Is there a reason not to name it `enableMigrate`?

Reply:
Good point @MicaiahReid. What do you think of `guardMigrate`, or `preventConsoleLogMigration` ?

Reply:
To me `preventConsoleLogMigration` is very clear. I wonder if there are other opinions, but I'd hate for this to get bogged down with this bike shedding.

Reply:
i like `preventConsoleLogMigration` as well